com.insynchq.Insync.yml

```diff
@@ -0,0 +1,49 @@
+app-id: com.insynchq.Insync
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: run.sh
+separate-locales: false
+rename-desktop-file: insync.desktop
+rename-icon: insync
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=network
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+    # Tray icon
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=org.kde.StatusNotifierItem
+  - --own-name=org.kde.*
+  # Needed for /dev/disk/by-id which insync accesses
+  - --device=all
+  - --filesystem=~/Insync
+  - --persist=.config
+  - --filesystem=xdg-config/autostart:create
+build-options:
+  strip: true
+modules:
+  - name: insync
+    buildsystem: simple
+    build-commands:
+      - ar x insync.deb
+      - tar xf data.tar.gz
+      - mv usr $FLATPAK_DEST
+      - cp -r $FLATPAK_DEST/usr/share $FLATPAK_DEST
+      - desktop-file-edit --set-key=Exec --set-value="run.sh start --no-daemon" $FLATPAK_DEST/share/applications/insync.desktop
+      - install -Dm644 com.insynchq.Insync.appdata.xml $FLATPAK_DEST/share/metainfo/com.insynchq.Insync.metainfo.xml
+      - install -Dm755 run.sh $FLATPAK_DEST/bin/run.sh
+    sources:
+      - type: file
+        dest-filename: insync.deb
+        only-arches:
+          - x86_64
+        url: https://cdn.insynchq.com/builds/linux/3.9.6.60027/insync_3.9.6.60027-plucky_amd64.deb
+        sha256: 5701d6019ca60a5349297c3512aa17e16cc81c7b793cdc541873961b1c96eaab
+      - type: file
+        path: com.insynchq.Insync.appdata.xml
+      - type: script
+        commands:
+          - exec /app/usr/lib/insync/insync %U
```

Comment:
what does `%U`  mean?

Reply:
Wrong placeholder should have been `"$@"`